src/plothist/plotters.py

```diff
@@ -995,6 +995,7 @@ def plot_data_model_comparison(
     model_uncertainty=True,
     model_uncertainty_label="Model stat. unc.",
     data_uncertainty_type="asymmetrical",
+    plot_only=None,
```

Comment:
I would put this argument after fig, ax_main, ax_comparison, so that these are defined and the explanation can be understood. We could also add that this argument can only be used when fig, ax_main and ax_comparison are not provided by the user.